cluster/mesos/docker/km/Dockerfile

```diff
@@ -14,4 +14,4 @@ RUN apt-get update -qq && \
     apt-get clean
 
 COPY ./bin/* /usr/local/bin/
-ADD ./opt/mesos-cloud.conf /opt/
+COPY ./opt/* /opt/
```

Comment:
Makes sense. Feel free to add it again with another small PR. In addition to mesos-cloud.conf we need the sandbox-overlay.tar.gz.